console/src/frame.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Coord {
+    pub x: i32,
+    pub y: i32,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Size {
+    pub height: f64,
+    pub width: f64,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Sound {
+    pub file_path: String,
+    pub can_play: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Sprite {
+    pub position: Coord,
+    pub size: Size,
+    pub is_hidden: bool,
+    pub sound: Option<Sound>,
```

Comment:
Sprite shouldn't have a sound, that's what the struct Sound is for.

Reply:
I put it because in the UML of the render handler, among the attributes of the Sprite, they said that a Sprite may or may not contain `Sound` so, if you wish, I will remove it. No problem

![image](https://github.com/Pending-Name-21/console/assets/97560564/7e1a44a5-7b40-4a42-a206-53a355d23bbb)

Reply:
oh, based on the merged UML, the idea is to handle a Sprite's sound through composition. So, at least for now, remove it.

Reply:
Done!